recipe/meta.yaml

```diff
@@ -1,40 +1,34 @@
-{% set name = "SQLAlchemy-Utils" %}
-{% set version = "0.33.5" %}
-{% set bundle = "tar.gz" %}
-{% set hash_type = "sha256" %}
-{% set hash = "d57246d50a54143140c097c6d80c3ac70c47396f44dffd263dc6c8024b3e367c" %}
+{% set version = "0.33.6" %}
 
 package:
-  name: {{ name|lower }}
+  name: sqlalchemy-utils
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.{{ bundle }}
-  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ bundle }}
-  {{ hash_type }}: {{ hash }}
+  url: https://pypi.org/packages/source/S/SQLAlchemy-Utils/SQLAlchemy-Utils-{{ version }}.tar.gz
+  sha256: 45ab41c90bfb8dd676e83179be3088b3f2d64b613e3b590187163dd941c22d4c
 
 build:
-  number: 0
-  script: python -m pip install --no-deps --ignore-installed .
+  number: 1000
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv"
 
 requirements:
-  build:
+  host:
     - python
     - pip
-
   run:
     - python
     - six
     - sqlalchemy >=1.0
+    # extras
     - anyjson >=0.3.3
     - babel >=1.3
     - arrow >=0.3.4
     - intervals >=0.7.1
     - phonenumbers >=5.9.2
     - passlib >=1.6,<2.0
     - colour >=0.0.4
-    - ipaddr  # [py2k]
-    - enum34  # [py<34]
```

Comment:
@lvoliveira and @igortg by removing the optional deps here we can make it `noarch`. However, if this is a problem for you we can restore these.

Reply:
No, it's not a problem